docs/source/api-proposal.rst

```diff
@@ -1,1000 +0,0 @@
-Calico API Design
```

Comment:
Yes, I think we should leave the 0MQ API description in place until we switch over.  Assuming we decide to actually publish the etcd stuff now (i.e. publish what Cory's done this week rather than leave it until we're ready to transition) then I agree with Neil that having a pointer in the 0MQ API doc to the new datastore description would be sensible.